### PR TITLE
Add sudoer_passwordless option.

### PR DESCRIPTION
**Pull request by chriszarate**
Not sure if there's a philosophical objection to this, but I need it for my environment.

### CODE REVIEW

**Comment by BrandonShutter** [comment]
I would use this.

**Comment by swalkinshaw** [comment]
@chriszarate yeah I'm not sure about this. I might have a philosophical objection but also concerned about adding options like this vs a single way of doing it.

**Comment by chriszarate** [comment]
Fair enough. Feel free to close.

**Comment by BrandonShutter** [comment]
Humor me, please.  Why is this a bad idea?  I personally have been using this.

**Comment by swalkinshaw** [comment]
@BrandonShutter I personally find typing the password a strong reminder for what is actually going on.

Might not actually be a valid reason though.

File: roles/users/tasks/main.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   lineinfile: dest=/etc/sudoers
               state=present
               regexp="^%sudo"
-              line="%sudo ALL=(ALL:ALL) ALL"
+              line="%sudo ALL={{ sudoer_passwordless | ternary('(ALL) NOPASSWD:', '(ALL:ALL)') }} ALL"
```
Comment (swalkinshaw) [inline roles/users/tasks/main.yml]: @chriszarate just realized an alternative solution I think might be better. Just have a variable called `sudo_group_permissions` which defaults to `(ALL:ALL)` and then you could customize it to `(ALL) NOPASSWD:`

**Comment by chriszarate** [comment]
Nice! That's much more flexible than the Boolean and ternary. I'll revise.

**Comment by swalkinshaw** [comment]
Closing this for now. @chriszarate feel free to re-open whenever you get a chance to work on this again.

**Pull request closed without merging**